src/features/island/fisherman/FishingChallenge.tsx

```diff
@@ -159,27 +159,21 @@ export const FishingChallenge: React.FC<Props> = ({
   const reel = () => {
     const greenBarAngle = extractAngle(greenBarProps.transform.get());
 
-    let tentacleAngle = extractAngle(tentacleProps.transform.get());
-
-    // Bar width / Circumference = 20 / (3.14159 * 200)
-    const fishBarOffset = 3.183;
-
-    tentacleAngle += fishBarOffset;
+    const tentacleAngle = extractAngle(tentacleProps.transform.get());
 
     const degreeDifference = Math.abs(
       ((normaliseAngle(greenBarAngle) - normaliseAngle(tentacleAngle) + 180) %
         360) -
         180
     );
 
-    const fishBuffer = 5;
+    const fishBuffer = 10;
```

Comment:
Fish is approx 20px, so this is half of the fish buffer on either side of the green line.